src/OpenTracing/Tracer.php

```diff
@@ -35,7 +35,7 @@
      */
     public function startSpan(
         $operationName,
-        SpanReference $parentReference = null,
+        Reference $parentReference = null,
         $startTimestamp = null,
         array $tags = []
     );
```

Comment:
Makes sense, we have a strong `SpanOptions` class which deals with all the complexity of the arguments. I am up to remove that.